tests/customer-resource-show-visibility-test.js

```diff
@@ -44,15 +46,20 @@ describeApplication('CustomerResourceShowVisibility', () => {
       });
     });
 
-    it('does not display the hidden/reason section', () => {
+    it('displays the hide/show toggle with show', () => {
```

Comment:
Alternate suggestion for this label: `displays the visibility toggle as switched to on`. Open to any other ideas that might make it more clear.